integrations/access/opsgenie/app.go

```diff
@@ -277,32 +318,26 @@ func (a *App) onDeletedRequest(ctx context.Context, reqID string) error {
 
 // Get services to notify from both annotations: /notify-services and /teams
 // Return error if both are empty
-func (a *App) getNotifyServiceNames(ctx context.Context, req types.AccessRequest) ([]string, error) {
+func (a *App) getNotifyServiceNames(ctx context.Context, req types.AccessRequest) ([]string, []string, error) {
```

Comment:
It's not clear what each of the return values are. Also, the method is named "get service names" but it now returns more than that.
```suggestion
func (a *App) getNotifyServicsAndTeams(ctx context.Context, req types.AccessRequest) (services []string, teams []string, error) {
```